tests/Tests/Extensions/NameValueCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Collections.Specialized;
+using Elasticsearch.Net.Extensions;
+using FluentAssertions;
+using Xunit;
+
+namespace Tests.Extensions
+{
+	public class NameValueCollectionExtensionsTests
+	{
+		[Theory]
```

Comment:
For regular `Fact` tests I have just noticed that there is an Elastic.Elasticsearch.Xunit library and `I` class. I'm not sure if this format is supported?

Reply:
Our `Tests` project can be run in both integration and unit test mode. This to avoid duplicating ceremony around testing API's. To annotate which assertions need to run when we use `[I]` and `[U]`, these do not support `MemberData` though. 

So technically this test will run in both contexts now. For now this is fine, long term we are likely in need to split of `Tests` and reserve the `[I]` and `[U]` tests for API level tests.